src/components/DetailsOverlay/Toggles/CaseSensitiveToggle/index.tsx

```diff
@@ -1,18 +1,14 @@
-import { QueryParams } from "constants/queryParams";
-import { useQueryParam } from "hooks/useQueryParam";
+import { useLogContext } from "context/LogContext";
 import BaseToggle from "../BaseToggle";
 
 const CaseSensitiveToggle: React.FC = () => {
-  const [caseSensitive, setCaseSensitive] = useQueryParam(
-    QueryParams.CaseSensitive,
-    false
-  );
+  const { searchState, setCaseSensitive } = useLogContext();
```

Comment:
Is it intentional that this toggle no longer updates the URL params (i.e. do we want case sensitivity to be shareable for search)?

Reply:
Yes it is i think it shouldn't be we don't share search results and since case sensitivity only affects search it doesn't make sense to share it.